src/app/blog/[title]/_components/GenerateTOC.tsx

```
@@ -0,0 +1,123 @@
+              if (targetElement) {
+                const yOffset = window.innerHeight * 0.2;
+                const y = targetElement.getBoundingClientRect().top + window.scrollY - yOffset;
+                window.scrollTo({ top: y, behavior: "smooth" });
+              }
+            }}
+          >
+            <a
+              href={`#${id}`}
+              className={aClass}
+              aria-current={selectedIndex === index ? "true" : undefined}
+              aria-label={`목차 항목: ${text}`}
```

Biome

[error] The HTML element with the interactive role "menuitem" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "menuitem".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)